manifest.php

```diff
@@ -32,7 +32,7 @@
     'label' => 'Generis Core',
     'description' => 'Core extension, provide the low level framework and an API to manage ontologies',
     'license' => 'GPL-2.0',
-    'version' => '13.6.0',
+    'version' => '13.7.0',
```

Comment:
Due to a breaking change, mentioned above, please consider bumping the major version.

```suggestion
    'version' => '14.0.0',
```